src/reforms/remnant/companions.txt

```diff
@@ -9,6 +9,9 @@ governments:remnant:reform_levels:1 += new companion_remnant = {
 	potential = {
 		has_reform = companion_remnant
 	}
+	custom_attributes = {
+		special_election = yes
```

Comment:
what is this?

Reply:
Basically you give this attribute to your government if it has a special election event instead of using elections.2